core-services/prow/02_config/stolostron/console/_prowconfig.yaml

```diff
@@ -3,9 +3,139 @@ branch-protection:
     stolostron:
       repos:
         console:
-          required_pull_request_reviews:
-            dismiss_stale_reviews: true
-            required_approving_review_count: 2
+          branches:
+            release-2.9:
+              protect: true
+              required_pull_request_reviews:
+                dismiss_stale_reviews: true
+                required_approving_review_count: 2
+              required_status_checks:
+                contexts:
+                - ci/prow/check
+                - ci/prow/images
+                - ci/prow/pr-image-mirror
+                - ci/prow/unit-tests-sonarcloud
+                - dco
+                - tide
+            release-2.10:
+              protect: true
+              required_pull_request_reviews:
+                dismiss_stale_reviews: true
+                required_approving_review_count: 2
+              required_status_checks:
+                contexts:
+                - ci/prow/check
+                - ci/prow/images
+                - ci/prow/pr-image-mirror
+                - ci/prow/unit-tests-sonarcloud
+                - dco
+                - tide
+            release-2.11:
+              protect: true
+              required_pull_request_reviews:
+                dismiss_stale_reviews: true
+                required_approving_review_count: 2
+              required_status_checks:
+                contexts:
+                - Red Hat Konflux / console-acm-211-on-pull-request
+                - Red Hat Konflux / console-mce-mce-26-on-pull-request
+                - Red Hat Konflux / enterprise-contract-acm-211 / console-acm-211
+                - Red Hat Konflux / enterprise-contract-mce-26 / console-mce-mce-26
+                - ci/prow/check
+                - ci/prow/images
+                - ci/prow/pr-image-mirror
+                - ci/prow/pr-image-mirror-mce
+                - dco
+                - tide
```

Comment:
It's fine to leave these here, but Prow will also detect its own jobs and mark them required if you want to leave them off.

Reply:
Yeah, let's keep it simple and only list Konflux jobs.